packages/react-app/src/components/views/quest-list.tsx

```diff
@@ -139,7 +141,7 @@ export default function QuestList() {
         <Dashboard />
         {!below('medium') && <Piggy />}
       </LineStyled>
-      <FilterWrapperStyled theme={currentTheme} isSmallResolution={below('medium')}>
```

Comment:
That props dont exist in that specif component but have another component with the same name has that props but different inner css